README.md

```diff
@@ -1,7 +1,38 @@
 # soccer-stonks
-or just call it STONKSS
+or just call it RON STONKSS
 
 ![Alt text](imgs/rondoGuy.png)
+
+## Setting up the Virtual Enviroments
+
+ **Create a Virtual Environment:**
+    ```sh
+    python -m venv venv
+    ```
+### Windows
+
+ **Activate the Virtual Environment:**
+    ```sh
+    .\venv\Scripts\activate
+    ```
+
+### macOS
+
+**Activate the Virtual Environment:**
+    ```sh
+    source venv/bin/activate
+    ```
+
+### Installing Dependencies
+
+Once the virtual environment is activated, install the required dependencies:
+```sh
+pip install -r requirements.txt
+```
+
+- To run kedro `kedro run`
+- To run streamlıt `streamlit run streamlit_app.py`
+
 ## Overview
 
```

Comment:
Nice work Hasan, can you also remove these explanations that kedro automatically created

Reply:
I did the changes.